src/pages/PassportRedemption/RedemptionClaimScreen.tsx

```diff
@@ -132,7 +135,10 @@ const PassportRedemptionClaim = ({ setCurrentScreenView }: Props) => {
           <ContentContainer>
             <CardText size="15px" color="#a8192e">
               Offer must be used in:
-              <span style={{fontWeight:'bold'}}> {formatTime(timeLeft)}</span>
+              <span style={{ fontWeight: 'bold' }}>
```

Comment:
You can also just wrap this in <strong></strong> or give it a separate class

Reply:
If you want the inline block behavior you can still wrap it in a span

Reply:
ill add this rq